acceptance/tests/windows/prevent_password_changes.rb

```diff
@@ -0,0 +1,104 @@
+require 'date'
```

Comment:
Are these acceptance tests worth it? Should these be unit tests instead?

Reply:
I think having them as unit tests and then having one acceptance test just to see if a warning is printed would be nice. Don't know if there's some stigma on doing expectations around logging in unit tests though.